datafusion-expr/src/expr.rs

```diff
@@ -228,6 +228,8 @@ pub enum Expr {
     },
     /// Represents a reference to all fields in a schema.
     Wildcard,
+    /// Represents a reference to all fields in a specific schema.
```

Comment:
I personally have a slight preference for a single `Expr::Wildcard` with optional qualifier

```rust
  Wildcard { qualifier: Option<String>}
```

but that is a personal preference and there is nothing wrong with this approach